forum/projects/utils.py

```diff
@@ -0,0 +1,15 @@
+def get_changed_fields(old_instance, new_instance):
+    """
+    Function compare project data and return changed
+    """

```

Comment:
Expand the docstring to include descriptions of the arguments and return value. This makes the function easier to understand and use.

Reply:
done!